### DISCUSSION TITLE
Explicit operator for DuckDBInterval to TimeSpan

### DISCUSSION THREAD

**Discussion by Seddryck** [Giorgi/DuckDB.NET#112] [General]
I was wondering if it would make sense for you if I'd submit a PR to add an explicit operator for DuckDBInterval to transform it into a TimeSpan. I won't implement an implicit operator as the DuckDBInterval is richer than the TimeSpan type (regarding the capability to define an interval based on months/years). 

I could also add the reverse operators (from TimeSpan to DuckDBInterval) and in that case, both the implicit and explicit operators.

**Comment by Giorgi** [comment]
What do you plan to do in cases when the value cannot be converted to `TimeSpan`?

**Reply by Seddryck** [reply]
Throw an exception (Argument out-of-range), that's why it should be explicit.

**Reply by Giorgi** [reply]
@Seddryck I think that makes sense.
